tests/cupy_tests/cuda_tests/test_cub.py

```diff
@@ -0,0 +1,159 @@
+import functools
+import unittest
+
+import cupy
+from cupy import testing
+
+
+if cupy.cuda.cub_enabled is False:
```

Comment:
Could you annotate the class using `unittest.skipIf` whenever possible?
Module-level code runs during import and often cause problems that is hard to track...

Reply:
OK. Done.